packages/modules/json/bat.py

```diff
@@ -32,14 +32,17 @@ def __init__(self, device_id: int, component_config: dict) -> None:
 
     def update(self, response) -> None:
         config = self.component_config["configuration"]
+        if config["jq_power"] is not None:
+            power = jq.compile(config["jq_power"]).input(response).first()
+        else:
+            power = 0
```

Comment:
```suggestion
        power = jq.compile(config["jq_power"]).input(response).first()
```
Die Leistung des Speichers muss ausgelesen werden, da dieser sonst nicht in der Regelung berücksichtigt werden kann. `None` ist der default-Value, der vom Benutzer überschrieben werden muss.